lib/backend/model/ippool.go

```diff
@@ -93,6 +93,7 @@ func (options IPPoolListOptions) KeyFromDefaultPath(path string) Key {
 type IPPool struct {
 	CIDR          net.IPNet `json:"cidr"`
 	IPIPInterface string    `json:"ipip"`
+	IPIPOffSubnet bool      `json:"ipip_off_subnet"`
```

Comment:
If we're just using the IPv4/6Network fields on the Node.BGP object, then do we need this at all?

I think I'd argue for this being explicit anyway.